dbt_project.yml

```diff
@@ -1,18 +1,18 @@
 name: 'linkedin'
-version: '0.12.0'
+version: '1.0.0'
 config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 vars:
   linkedin:
-    ad_analytics_by_creative: "{{ ref('stg_linkedin_ads__ad_analytics_by_creative') }}"
-    creative_history: "{{ ref('stg_linkedin_ads__creative_history') }}"
-    campaign_history: "{{ ref('stg_linkedin_ads__campaign_history') }}"
-    campaign_group_history: "{{ ref('stg_linkedin_ads__campaign_group_history') }}"
-    account_history: "{{ ref('stg_linkedin_ads__account_history') }}"
-    ad_analytics_by_campaign: "{{ ref('stg_linkedin_ads__ad_analytics_by_campaign') }}"
-    geo: "{{ ref('stg_linkedin_ads__geo') }}"
-    monthly_ad_analytics_by_member_country: "{{ ref('stg_linkedin_ads__monthly_ad_analytics_by_country') }}"
-    monthly_ad_analytics_by_member_region: "{{ ref('stg_linkedin_ads__monthly_ad_analytics_by_region') }}"
+    ad_analytics_by_creative: "{{ source('linkedin_ads', 'ad_analytics_by_creative') }}"
+    creative_history: "{{ source('linkedin_ads', 'creative_history') }}"
+    campaign_history: "{{ source('linkedin_ads', 'campaign_history') }}"
+    campaign_group_history: "{{ source('linkedin_ads', 'campaign_group_history') }}"
+    account_history: "{{ source('linkedin_ads', 'account_history') }}"
+    ad_analytics_by_campaign: "{{ source('linkedin_ads', 'ad_analytics_by_campaign') }}"
+    geo: "{{ source('linkedin_ads', 'geo') }}"
+    monthly_ad_analytics_by_member_country: "{{ source('linkedin_ads', 'monthly_ad_analytics_by_country') }}"
+    monthly_ad_analytics_by_member_region: "{{ source('linkedin_ads', 'monthly_ad_analytics_by_region') }}"
```

Comment:
Should we call out this change in the CHANGELOG in case people were utilizing the variables?